.github/workflows/license-headers.yml

```diff
@@ -3,14 +3,27 @@
 
 ---
 name: '[License] Check license headers'
-
 on:
-  pull_request:
-
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+      - bitnami:main
+# Remove all permissions by default
+permissions: {}
 jobs:
   license-headers-linter:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
     steps:
       - uses: actions/checkout@v3
+        name: Checkout Repository
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
IIRC, it isn't needed to add this and we usually omit it in out workflows right?

Reply:
In this case we are moving from `pull_request`, where the `ref` and the `repository` points to the forked repository , to [pull_request_target](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) where those properties points to the main branch and repository.